paddle/operators/pad_op.cc

```diff
@@ -111,11 +110,29 @@ class PadOpGrad : public framework::OperatorWithKernel {
   }
 };
 
+class PadOpGradMaker : public framework::SingleGradOpDescMaker {
+ public:
+  using framework::SingleGradOpDescMaker::SingleGradOpDescMaker;
+
+ protected:
+  std::unique_ptr<framework::OpDescBind> Apply() const override {
```

Comment:
I still think `Apply()` is not necessary. We can let op developers override `operator()` directly by themselves. The definition of `operator()` has dictated that the return value should be a `vector`.

Reply:
We can discuss later and change if necessary.